tools/merge-guard.py

```
@@ -156,11 +156,29 @@ def leg(name, ok, detail):
+        losers = []          # files this PR removes more from than it adds
         for line in stat.strip().splitlines():
             parts = line.split("\t")
             if len(parts) >= 2 and parts[0].isdigit() and parts[1].isdigit():
-                add += int(parts[0]); dele += int(parts[1])
-        leg("4 three-dot diff", add >= dele, f"+{add} -{dele} (net {add - dele:+d})")
+                # ⚠ NOT `a, d` — `d` is the PR dict in this scope. Shadowing it made
+                # `d.get("createdAt")` raise AttributeError on an int, and the paired
+                # suite caught it where --self-test could not: leg 5 is only reached
+                # when a PR record exists, which self_test() never builds.
+                ins, dels = int(parts[0]), int(parts[1])
+                add += ins; dele += dels
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)